component/azstorage/block_blob.go

```diff
@@ -1021,8 +1031,10 @@ func (bb *BlockBlob) removeBlocks(blockList *common.BlockOffsetList, size int64,
 		}
 
 	}
-
+	blk := blockList.BlockList[index]
```

Comment:
In the above section (lines [1023-1028](https://github.com/Azure/azure-storage-fuse/blob/ashruti/fixes/component/azstorage/block_blob.go#L1023-L1028)), we can skip the `ReadInBuffer` call.
`blk.Data` already has the block's data in it. So, we can just do something like,
`blk.Data = blk.Data[:blk.EndIndex-blk.StartIndex]`
Please confirm this part.

Reply:
`blk.Data` does not contain any data before calling ReadInBuffer so this method will not work.